src/test/regress/upgrade/config.py

```diff
@@ -0,0 +1,32 @@
+BEFORE_UPGRADE_SCHEDULE = './before_upgrade_schedule'
+AFTER_UPGRADE_SCHEDULE = './after_upgrade_schedule'
+
+
+class Config():
+    def __init__(self, arguments):
+        self.old_bindir = arguments['--old-bindir']
+        self.new_bindir = arguments['--new-bindir']
+        self.pg_srcdir = arguments['--pgxsdir']
+        self.temp_dir = './tmp_upgrade'
+        self.old_datadir = self.temp_dir + '/oldData'
+        self.new_datadir = self.temp_dir + '/newData'
+        self.settings = {
+            'shared_preload_libraries': 'citus',
+            'citus.node_conninfo': 'prefer'
```

Comment:
⚠️ This needs to be changed to fix sslmode, after that its ready to merge.

```python
  'citus.node_conninfo': 'sslmode=prefer'
```

The `node_conninfo` guc consists of key/value pairs that are parts of postrges' connection string: https://www.postgresql.org/docs/10/libpq-connect.html#LIBPQ-CONNSTRING